spec/intl.html

```diff
@@ -1721,6 +1776,7 @@ <h1>Temporal.Calendar.prototype.yearMonthFromFields ( _fields_ [ , _options_ ] )
               1. Let _fieldNames_ be CalendarDateFields(_calendar_.[[Identifier]], « *"month"*, *"monthCode"*, *"year"* »).
               1. Set _fields_ to ? PrepareTemporalFields(_fields_, _fieldNames_, « »).
               1. Let _overflow_ be ? ToTemporalOverflow(_options_).
+              1. Perform ! CreateDataPropertyOrThrow(_fields_, *"day"*, *1*<sub>𝔽</sub>).
```

Comment:
Can there ever be a case where a month has no date with `day===1` ?  AFAIK this isn't an issue for any current calendars, but in theory it could happen with some future calendar. 

Perhaps one way to address this would be to use `overflow: 'constrain'` in the call to `CalendarDateToISO` below, and then if the user's overflow was `reject`, then call it again with the day returned by the first call?

Reply:
I think we can do it more simply — something like "Let _firstDay_ be the 1-based index of the first day of the month described by _fields_ (i.e., 1 unless the month has skipped its first day)". It should be clear that this is always 1 unless it's a CLDR calendar that skips days.

Reply:
Sounds good to me.